### ISSUE TITLE
Remove explicit session.finish call

### ISSUE THREAD

**Issue by wenzeslaus** [veroandreo/foss4g2022_grass4rs#3]
The explicit call of _session.finish_ is not needed, remove it. Leaving it there means that after executing that cell, you can't get back to any other cell. Generally, you may want that behavior or you may want to align that with the standard non-Jupyter _init_ use (_grass.script[.setup].init_), but for workshop focused on remote sensing, I recommend greater flexibility.

[grass.jupyter.setup.init](https://grass.osgeo.org/grass82/manuals/libpython/grass.jupyter.html#grass.jupyter.setup.init) doc says:

> The session is ended when session.finish is called or when the object is destroyed when kernel ends or restarts. This function returns a copy of an internally kept reference, so the return value can be safely ignored when not needed.

The [jupyter_tutorial.ipynb](https://mybinder.org/v2/gh/OSGeo/grass/8.2.0?urlpath=lab%2Ftree%2Fdoc%2Fnotebooks%2Fjupyter_tutorial.ipynb) notebook says:

>  When the kernel for the notebooks shuts down or is restarted, the session ends automatically. The session can be explicitly ended using session.finish(), but that's usually not needed in notebooks.

In other words, explicit use of `session.finish()` is neither required nor considered a best practice according to the documentation. Feel free to extend it as needed.

**Comment by veroandreo** [comment]
Thanks for the heads-up @wenzeslaus! I believe I somehow followed this: https://github.com/OSGeo/grass/blob/main/doc/notebooks/scripting_example.ipynb. Now, if the function is not needed nor considered a good practice, when is it used then? Just curious.

In any case, this is WIP. The notebook here was an exercise I did for a course and we'll adapt for the workshop

**Comment by wenzeslaus** [comment]
As the [scripting_example.ipynb](https://github.com/OSGeo/grass/blob/main/doc/notebooks/scripting_example.ipynb) notebook says:

> This document is a Jupyter Notebook, but unlike the [jupyter_example.ipynb](https://github.com/OSGeo/grass/blob/d0b904f6e9077ee5f09f80e8ddbc9e7bb5b2c050/doc/notebooks/jupyter_example.ipynb) notebook, the examples in this notebook are meant to be used primarily outside of a notebook.

While the [jupyter_example.ipynb](https://github.com/OSGeo/grass/blob/main/doc/notebooks/jupyter_example.ipynb) notebook says:

> The grass.jupyter package simplifies the launch of GRASS GIS in Jupyter Notebook and provides several useful classes for creating, displaying, and saving GRASS GIS maps.

The conclusion from the two notebooks should be that the notebooks with _grass.jupyter_ are for notebooks and don't have an explicit _finish_ call while the notebook using _grass.script.setup.init_ is meant to be basis for generic Python code outside of notebooks and has an explicit _finish_ call. No harm done when _grass.jupyter_ is used with _finish_, but the behavior is sub-optimal.

**Comment by veroandreo** [comment]
`finish` call removed in https://github.com/veroandreo/foss4g2022_grass4rs/commit/a5178211c4624cae685f5ad0e59e3bb18ae9aadc
